packages/image_picker/image_picker_platform_interface/lib/src/types/picked_file/lost_data.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:cross_file/cross_file.dart';
```

Comment:
Let's put this lost_data outside of the picked_file subdirectory, now that it's not going to use picked_file. Creating a copy of the class in: `lib/src/types/lost_data.dart` is probably enough.